toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -129,89 +130,104 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
-			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
-		}
-	}
-
-	if len(failedSRPMsTests) != 0 {
-		logger.Log.Info("Failed SRPMs tests:")
-		for _, failure := range failedSRPMsTests {
-			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
+		keys := getSortedKeys(skippedSRPMsTests)
+		for _, skippedSRPMsTest := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(skippedSRPMsTest))
 		}
 	}
 
 	if len(blockedSRPMs) != 0 {
 		logger.Log.Info("Blocked SRPMs:")
-		for srpm := range blockedSRPMs {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+		keys := getSortedKeys(blockedSRPMs)
+		for _, blockedSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(blockedSRPM))
 		}
 	}
 
 	if len(blockedSRPMsTests) != 0 {
 		logger.Log.Info("Blocked SRPMs tests:")
-		for srpm := range blockedSRPMsTests {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+		keys := getSortedKeys(blockedSRPMsTests)
+		for _, blockedSRPMsTest := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(blockedSRPMsTest))
 		}
 	}
 
 	if len(unresolvedDependencies) != 0 {
 		logger.Log.Info("Unresolved dependencies:")
-		for dependency := range unresolvedDependencies {
-			logger.Log.Infof("--> %s", dependency)
+		keys := getSortedKeys(unresolvedDependencies)
+		for _, unresolvedDependency := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(unresolvedDependency))
 		}
 	}
 
 	if len(rpmConflicts) != 0 {
 		conflictsLogger("RPM conflicts with toolchain: ")
+		sort.Strings(rpmConflicts)
 		for _, conflict := range rpmConflicts {
 			conflictsLogger("--> %s", conflict)
 		}
 	}
 
 	if len(srpmConflicts) != 0 {
 		conflictsLogger("SRPM conflicts with toolchain: ")
+		sort.Strings(srpmConflicts)
 		for _, conflict := range srpmConflicts {
 			conflictsLogger("--> %s", conflict)
 		}
 	}
+
+	if len(failedSRPMs) != 0 {
+		logger.Log.Info("Failed SRPMs:")
+		keys := getSortedKeys(failedSRPMs)
+		for _, key := range keys {
+			failure := failedSRPMs[key]
+			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
+		}
+	}
+
+	if len(failedSRPMsTests) != 0 {
+		logger.Log.Info("Failed SRPMs tests:")
+		keys := getSortedKeys(failedSRPMsTests)
+		for _, key := range keys {
+			failure := failedSRPMsTests[key]
+			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
+		}
+	}
+
+	if len(prebuiltSRPMs) != 0 {
+		logger.Log.Info("Prebuilt SRPMs:")
+		keys := getSortedKeys(prebuiltSRPMs)
+		for _, prebuiltSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(prebuiltSRPM))
+		}
+	}
+
+	if len(builtSRPMs) != 0 {
+		logger.Log.Info("Built SRPMs:")
+		keys := getSortedKeys(builtSRPMs)
+		for _, builtSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(builtSRPM))
+		}
+	}
+
```

Comment:
Could we split the summary, (or maybe duplicate it?) here? It can be a pain to scroll all the way up to see if there is anything unexpected when you can just see "build x#, failed 0" and be happy.